java/ql/src/Security/CWE/CWE-524/SensitiveKeyboardCache.ql

```diff
@@ -0,0 +1,18 @@
+/**
+ * @name Android sensitive keyboard cache
+ * @description Sensitive information should not be saved to the keyboard cache.
+ * @kind problem
+ * @problem.severity warning
+ * @security-severity 8.1
+ * @id java/android/sensitive-keyboard-cache
+ * @tags security
+ *       external/cwe/cwe-524
+ * @precision high
```

Comment:
Given the heuristic nature of the query (the main logic being regex), I wonder if this should be `medium` instead. In your experiments, how does the FP ratio look like?

Reply:
It being `medium` would be consistent with the `CleartextStorage*` queries, which also handle sensitive data in a similar way.